R/rescale_catchments.R

```diff
@@ -38,7 +47,6 @@ rescale_characteristics <- function(vars, lookup_table) {
 #' "areasqkm." Used to retrieve adjusted catchment areas in the case of split
 #' catchments.
 #'
-#' @importFrom sf st_drop_geometry
```

Comment:
I don't see `sf::st_drop_geometry` used in `get_catchment_areas()` which is why I suggested omitting this line from the documentation.